src/components/DevTools/DevTools.delegations.tsx

```diff
@@ -1,8 +1,5 @@
-import {
-  BlockchainUtils,
-  DelegationNode,
-  DelegationRootNode,
-} from '@kiltprotocol/sdk-js'
+import { DelegationNode, DelegationRootNode } from '@kiltprotocol/sdk-js'
+import { BlockchainUtils } from '@kiltprotocol/chain-helpers'
```

Comment:
Is `BlockchainUtils` not exported from `@kiltprotocol/sdk-js`?

Reply:
I tried importing it but couldn't find `BlockchainUtils` in `@kiltprotocol/sdk-js`.

Reply:
it is supposed to be there...

Reply:
Yes, since we `submitSignedTx` and `IS_IN_BLOCK`.

Reply:
Updated it now to fix this issue.